src/niscope/templates/session.py/get_equalization_filter_coefficients.py.mako

```diff
@@ -5,6 +5,9 @@
 
     suffix = method_template['method_python_name_suffix']
 %>\
+% if f['use_session_lock']:
```

Comment:
These three lines are in (almost?) every method template. Can't we do this at the session level, before we render the method template?

Reply:
Moved check into session.py.mako (non nimodinst)

This also allowed removing per driver `use_session_lock`. Always set to `True` unless already set. nimodinst session.py.mako doesn't use it and never add locking.